exampleCourse/questions/demo/workspace/rstudio/question.html

```diff
@@ -0,0 +1,8 @@
+<pl-question-panel>
+<markdown>
+You can press the button below to launch an RStudio workspace that will persist for students, even if they refresh this page and click "Open workspace" again, because the question has been configured as "single variant" in `info.json`.
+
+**Note for instructors**: If you are an instructor previewing the question *in staff mode*, then refreshing this page and clicking the "Open workspace" button *will* take you to a fresh workspace container. Preview in student mode to test the persistence behavior. You can do this by going to the most recent instance of the example course, changing your user mode in the upper right to "Student view without access restrictions," and opening the "Questions using workspaces" assessment.
```

Comment:
Other demo workspace questions don't have this information; I'd be inclined to omit it for consistency.

This information is also not 100% correct; if you click "save" or "save and grade", the variant ID is stored in the URL, and the same workspace instance will be used.